### PR TITLE
Export custom scripts reference

### PR DESCRIPTION
**Pull request by JoneLau**

### CODE REVIEW

**Comment by JoneLau** [comment]
update cocos-creator/engine-3d#242

File: Assets/Editor/utils.cs [hunk 1]
```diff
@@ -575,5 +575,29 @@ public class Utils {
...
     }
+
+    public static string refPathLookup(GameObject go, GameObject root) {
+      string path = "/" + go.name;
+      Transform parent = go.transform;
+
+      while (true) {
+        parent = parent.parent;
+
+        if (root==null) {
```
Comment (jwu) [inline Assets/Editor/utils.cs]: if ( root == null && parent == null ) {
}

File: Assets/Editor/utils.cs [hunk 2]
```diff
@@ -575,5 +575,29 @@ public class Utils {
 
       return new string[] { "center", "center" };
     }
+
+    public static string refPathLookup(GameObject go, GameObject root) {
+      string path = "/" + go.name;
```
Comment (jwu) [inline Assets/Editor/utils.cs]: path = go.name; 不需要 /

File: Assets/Editor/utils.cs [hunk 3]
```diff
@@ -575,5 +575,29 @@ public class Utils {
...
+          if (parent == null) {
+            break;
+          }
+        } else {
+          if (parent.gameObject == root) {
+            break;
+          }
+        }
+
+        path = "/" + parent.name + path;
```
Comment (jwu) [inline Assets/Editor/utils.cs]: parent.name + "/" + path;

**Pull request merged**